### PR TITLE
Gfrenkel/sql columns option

### PR DESCRIPTION
**Pull request by Gil-Frenkel**
# Add column names to SQLDatabase.run() result

Description:
Current state: SQLDatabase.run() method returns a result from the database after executing a SQL query but there is no option of adding the column names as well to the returned result.

PR offer the following solution:

Adding a argument "include_column_names_on_query" to the SQLDatabase constructor.
By default, this argument will be set to False, but when set to True, SQLDatabase.run() method will add the column names to the result (if the result is not empty).

Would love to hear some feedback!

  -  @hwchase17
  - @vowelparrot

### CODE REVIEW

File: langchain/sql_database.py [hunk 1]
```diff
@@ -37,6 +37,7 @@ def __init__(
         metadata: Optional[MetaData] = None,
         ignore_tables: Optional[List[str]] = None,
         include_tables: Optional[List[str]] = None,
+        ignore_column_names_on_query: bool = True,
```
Comment (hwchase17) [inline langchain/sql_database.py:40]: marginal preference to have this opt in and default to False, eg `include_column_names_on_query = False`

Comment (Gil-Frenkel) [reply langchain/sql_database.py]: Done.

**Comment by hwchase17** [comment]
i tried to resolve merge conflicts with master, i think i did it okay?

**Comment by Gil-Frenkel** [comment]
@hwchase17 Yup! Looks great!

**Comment by Gil-Frenkel** [comment]
@hwchase17 Yup! Looks great! Had to make an important fix.. see last commit (dumb mistake on my end)

**Review by hwchase17 — APPROVED** [review]
thanks!

**Comment by Gil-Frenkel** [comment]
@hwchase17 There was a formatting issue in the lint workflow fixed that (last commit)

**Comment by leo-gan** [comment]
@Gil-Frenkel Hi , could you, please, resolve the merging issues? After that ping me and I push this PR for the review. Thanks!

**Comment by efriis** [comment]
Closing because the PR wouldn't line up with the current directory structure of the library (would need to be in /libs/langchain/langchain instead of /langchain). Feel free to reopen against the current head if it's still relevant!

**Pull request closed without merging**